### ISSUE TITLE
Feature Request: built-in current, edition selectors (and maybe tags)

### ISSUE THREAD

**Issue by panoptical** [codex-press/app#3]
It would be great if the metadata section contained built-in Edition and Current fields, preferably with selectors that would let the user pick from a menu (like a drop-down list) populated with existing Editions and Currents.

As of now the user has to type the exact current name in each new article.  A typo would cause the article not to appear in its correct current.  Also the user has to add the "current" metadata field manually.

A "tags" field built-in might also be useful.  We don't tag posts as of now but other CMSes use tags as a way of organizing and providing additional connectedness to articles.

**Comment by twfarnam** [comment]
Since Codex is designed for various users, we can't have any Coda specific meta information. A possible solution would be to add custom metadata fields and possible values to the Coda account, but it will be tricky.

I'm going to give this a lower priority since there's a solution that works for you guys currently (just spelling it correctly). Although if it ends up being very nettlesome for the team, please flag it again!

**Comment by panoptical** [comment]
I agree with low priority.  As for implementation, what about having space for user-defined metadata fields (and/or user-defined article templates)?

**Comment by twfarnam** [comment]
Yeah, love it. I could see a section in the Account page (https://codex.press/edit/account) that would let you add custom metadata fields to articles.

Right now you guys actually can edit the one template that shows up when you create a new story: https://codex.press/edit/article/coda/template

The ideal solution would involve some sort of validation that ensures people fill in all the required metadata. That could get pretty aggravating if implemented poorly though.
